Gemfile

```diff
@@ -193,7 +193,7 @@ group :development, :test do
   # Test framework
   gem 'rspec-rails', '7.1.0'
 
-  gem 'selenium-webdriver'
+  gem 'capybara-playwright-driver'
```

Comment:
🤔 I don't think we've committed to this change. Can we remove the playwright stuff for now please?